package.json

```diff
@@ -47,11 +47,13 @@
     "jest-cli": "^27.4.5",
     "prettier": "~2.7.0",
     "puppeteer": "^19.4.1",
+    "rollup-plugin-dotenv": "^0.5.0",
     "scss-bundle": "^3.1.1",
     "typescript": "^4.9.0"
   },
   "license": "MIT",
   "dependencies": {
+    "@genexus/reporting-api": "^0.1.0",
```

Comment:
Since we didn't add the package-lock.json file in this PR, my understanding is that we should use the `"^0.1.2"` version.

Reply:
you are right. I updated the version